contrib/completion/powershell/posh-docker.psm1

```diff
@@ -0,0 +1,179 @@
+# Powershell completion for docker
```

Comment:
@duglin no this reverts that; also we're not going to be copying that completion, just referring to it if someone wants completion scripts